.travis.yml

```diff
@@ -108,9 +108,9 @@ script:
     - /tmp/julia/bin/julia --precompiled=no -e 'true' &&
         /tmp/julia/bin/julia-debug --precompiled=no -e 'true'
     - /tmp/julia/bin/julia -e 'versioninfo()'
-    - export JULIA_CPU_CORES=2 && export JULIA_TEST_MAXRSS_MB=600 && cd /tmp/julia/share/julia/test &&
-        /tmp/julia/bin/julia --check-bounds=yes runtests.jl $TESTSTORUN &&
-        /tmp/julia/bin/julia --check-bounds=yes runtests.jl libgit2-online pkg
+    - export JULIA_CPU_CORES=2 JULIA_TEST_MAXRSS_MB=600 &&
+        /tmp/julia/bin/julia --check-bounds=yes -e "Base.runtests(\"$TESTSTORUN\", \"$JULIA_CPU_CORES\")" &&
```

Comment:
I have stared at the build files for too long. `$JULIA_CPU_CORES` should not be a string of course.